docs/content/backup_restore.md

```diff
@@ -1,9 +1,130 @@
 
+The backup command will use [predefined or configured binary](./configuration#binaries) for your database vendor with correct parameters
+to dump your database.
+
+Each time you launch the backup command, [backup file is store in a directory](./configuration#storage-directory) (See
+[Storage section](#storage) below for more information on how backup files are stored).
+
+With time, this directory will grow, that's why a [backup expiration age](./configuration#storage-directory#backup-expiration-age)
+was added. Every time you launch the command, at the end, it will be asked if you want to remove obsolete
+backup files (ie files that have pasted their expiration date).
+
+```sh
+console db-tools:backup
+```
+You can specify the behaviour of the command with some options.
+
+### Connection
+
+By default, the command will backup the database from the default DBAL connection.
+
+You can choose to backup a database from another connection with `--connection` option:
+
+```sh
+console db-tools:backup --connection other_connection_name
+```
+
+### Excluded tables
+
+You may have configured [tables to be excluded in the bundle configuration](./configuration#excluded-tables).
+If so, these tables will be automatically excluded each time you launch the command.
+
+But if you want to temporary exclude some tables, run the command with the `--excluded-table` option:
+
+
+```sh
+# Exclude a table
+console db-tools:backup --excluded-table table_to_exclude
+
+# Or more
+console db-tools:backup --excluded-table table_to_exclude_1 --excluded-table table_to_exclude_2
+```
+
+### No cleanup
+
+If you want to skip the cleanup step, launch it with option `--no-cleanup':
+
+```sh
+console db-tools:backup --no-cleanup
+```
+
+:::warning
+Note that using this option, backup files will never be cleaned up
+:::
+
 ## Restore command
 
-## Storage
\ No newline at end of file
+The restore command will use [predefined or configured binary](./configuration#binaries) for your database vendor with correct parameters
+to restore your database from existing backup files.
+
+
+```sh
+console db-tools:restore
+```
+
+You can specify the behaviour of the command with some options.
+
+### Connection
+
+By default, the command will restore the database from the default DBAL connection.
+
+You can choose to restore a database from another connection with `--connection` option:
+
+```sh
+console db-tools:restore --connection other_connection_name
+```
+
+### Filename
+
+When you launch this command, existing backup files will be listed and it
+will be asked you to choose one to restore.
+
+If you want to skip this step, or if your backup file is unknown to the DbToolsStorage,
+you can specify a file to restore with the `--filename` option:
+
+```sh
+console db-tools:restore --filename /path/to/my/backup.sql
+```
+
+### Force
+
+Each time you ru this command, as it is not a sensitive operation, a confirmation will
+be asked. If you want to skip it, use the `--force` option.
```

Comment:
I wonder if it's not the role of the default `--no-interaction` Symfony console option?

Reply:
I should have read a bit more before commenting. There's a security for the production environment. That's not the case with `--no-interaction`.